src/main/java/com/devathon/preguntonic/model/Player.java

```diff
@@ -0,0 +1,45 @@
+package com.devathon.preguntonic.model;
+
+import java.util.UUID;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class Player {
+    
+    private UUID id;
+    private boolean isAdmin;
+    private String ipAddress;
+    private String avatar; // TODO: Change to enum if needed
+    private String nickname;
+    private PlayerStatus status;
+    private int score;
+
+    // Transient (lobby)
+    private boolean isReady;
+
+    // Transient (game)
+    private boolean responded;
+    private long responseTime;
+    private UUID responseId;
+    private boolean readyForNextQuestion;
```

Comment:
NIT: in a future we can extract this parameter to other class due it is more related with game logic

Reply:
Yes... i think we should maintain those params until MVP is released